src/witan/app/test_data.clj

```diff
@@ -310,13 +382,27 @@
                                                           "Low Fertility"]}]
                               :input-data [development-category]
                               :output-data [output-category]
-                              :fixed-input-data [(data/->Data population-data)
-                                                 (data/->Data institutional-data)
-                                                 (data/->Data private-housing-data)
-                                                 (data/->Data households-data)
-                                                 (data/->Data dwellings-data)
+                              :fixed-input-data [(data/->Data ward-php)
+                                                 (data/->Data ward-total-popn-baseyear)
+                                                 (data/->Data ward-fertility-rates)
+                                                 (data/->Data ward-survival-rates)
+                                                 (data/->Data ward-births)
+                                                 (data/->Data ward-deaths)
+                                                 (data/->Data ward-inmigration)
+                                                 (data/->Data ward-outmigration)
+                                                 (data/->Data ward-census-dwellings)
+                                                 (data/->Data ward-institutional-data)
                                                  (data/->Data past-development-data)
-                                                 (data/->Data future-development-data)]}
+                                                 (data/->Data future-development-data)
+                                                 (data/->Data high-fert-principal-births-data)
+                                                 (data/->Data high-fert-principal-deaths-data)
+                                                 (data/->Data high-fert-principal-sya-data)
+                                                 (data/->Data standard-fert-principal-births-data)
+                                                 (data/->Data standard-fert-principal-deaths-data)
+                                                 (data/->Data standard-fert-principal-sya-data)
+                                                 (data/->Data low-fert-principal-births-data)
+                                                 (data/->Data low-fert-principal-deaths-data)
+                                                 (data/->Data low-fert-principal-sya-data)]}
```

Comment:
These principal migration births, deaths and sya feed into the trend-based ward model only